src/api/uploads/MultiputPart.js

```diff
@@ -26,38 +30,299 @@ class MultiputPart extends Base {
         | typeof PART_STATE_UPLOADED;
     timing: Object;
     uploadedBytes: number;
+    uploadUrl: string;
+    onProgress: Function;
+    onSuccess: Function;
+    onError: Function;
+    data: Object;
```

Comment:
Is it possible to use a more descriptive type for this variable?

Reply:
done